stl/inc/xutility

```diff
@@ -3226,6 +3211,19 @@ public:
     using pointer         = _Iter;
 
 #ifdef __cpp_lib_concepts
+    constexpr static auto _Get_iter_concept() {

```

Comment:
Nit: is there a convenient `__cpp_lib_concepts` section in a `private` part of the class we could move this into? We try (but are not completely consistent about) to avoid exposing even `_Ugly` names in `public`.